src/test/java/template/neodymium/tests/smoke/HomePageTest.java

```diff
@@ -6,15 +6,14 @@
 
 import template.flows.OpenHomePageFlow;
 import template.neodymium.tests.AbstractTest;
-import template.pageobjects.pages.HomePage;
 
 public class HomePageTest extends AbstractTest
 {
     @Test
     public void testVisitingHomepage()
     {
         // Goto the home page
-        HomePage homePage = OpenHomePageFlow.flow();
+        var homePage = OpenHomePageFlow.flow();
 
         // short validation to check that the correct page was opened, should be moved to OpenHomePageFlow
         homePage.isExpectedPage();
```

Comment:
move this line to `OpenHomePageFlow.flow()` 
add another comment to the call above stating that a short validation is performed automatically

Reply:
Done